README.md

```diff
@@ -7,13 +7,14 @@
 BlaBlaCar.com and Spotify.com) and most of the [popular PHP projects][2] (including
 Drupal and Magento).
 
+Symfony follows the [semantic versioning][5] strictly, publishes "Long Term
+Support" (LTS) versions and has a [release process][6] that is predictable and
+business-friendly.
```

Comment:
I would instead remove this paragraph as we don't want to highlight the LTS.

Reply:
@fabpot hello, thanks for your comment

according to https://symfony.com/what-is-symfony

```
Symfony Philosophy
Embracing and promoting professionalism, best practices, standardization and interoperability of applications.
```

uppon the `professionalism` word
imho it is important to mention on the readme of such project :)
so may i ask you why LTS is going to be removed?

cheers

Reply:
Relying on LTS means that you only upgrade every 2 years. That's a LOT of work. Upgrading every 6 months is much easier. Now that Symfony is becoming quite good at not breaking BC, continuously upgrading seems a much better strategy to me.

Reply:
Oki understood
I will continue this soon